motion/motionprocessor.go

```diff
@@ -132,7 +140,27 @@ func (mp *MotionProcessor) GetRecentFrame(frame *lepton3.Frame) *lepton3.Frame {
 	return mp.frameLoop.CopyRecent(frame)
 }
 
+// setSunriseSunsetWindow sets the recording window based of todays sunset and sunrise location
+func (mp *MotionProcessor) setSunriseSunsetWindow() {
+	if mp.sunriseSunsetWindow {
+		curTime := time.Now()
+		if mp.nextSunriseCheck.Before(curTime) {
+			delay := time.Duration(mp.sunriseSunsetDelay) * time.Minute
+			location := curTime.Location()
+			year, month, day := curTime.Date()
+
+			rise, set := sunrise.SunriseSunset(mp.locationConfig.Latitude, mp.locationConfig.Longitude, year, month, day)
+			rise = rise.Add(delay)
+
+			set = set.Add(delay)
```

Comment:
I would have expected the "delay" to subtract from the sunrise time and add to sunset time. In this case, "delay" isn't the best name for the value. Perhaps "padding" or "offset" (in the config file too)